server/routes/index.js

```diff
@@ -1,9 +1,11 @@
 import mockAndelAPI from '../mockAndelaApi';
 import automationsAPI from './automationRouter';
 import workerStatus from '../controllers/workerStatus';
+import createAutoPlacements from '../seeder';
 
 export default (app) => {
   app.get('/worker-status', workerStatus);
   app.use('/mock-api/api/v1', mockAndelAPI);
   app.use('/api/v1/automations', automationsAPI);
+  app.use('/api/v1/auto', createAutoPlacements);
```

Comment:
I now see what you're doing. Great initiative. What if we dont create an endpoint for this? instead invoke the method/function to execute `createAutoPlacements` directly upon app startup? Making an http request to an endpoint within the app itself looks inefficient. Just call the function

Reply:
I did this initially and it wasn't working but I think I have resolved what was the issue back then. Will update this.